timer.js

```diff
@@ -1,22 +1,17 @@
 function startTimer(durationSeconds = 25 * 60) {
-  if (!state.currentTodo) {
-    alert("Chose todo from table");
-    return;
-  }
-  clearInterval(state.timerIntervalId);
+
+  if (state.currentTodo && state.isWorkInterval) {
```

Comment:
when we have 2+ booleans in an if() statement, personally I like to make an "explanatory constant", its name will explain what we are checking for here and what is about to happen, easier to read!

```suggestion
const shouldStartTimer = state.currentTodo && state.isWorkInterval // only start the timer if we have selected an item (not sure what "isWorkInterval" means)
  if (shouldStartTimer) {
```

Reply:
I delete this statement, i found it was not necessary and use this explanatory constant in another case:   
![image](https://github.com/antondurinin/ToDo/assets/167963627/57e0683f-2661-4252-85fe-b81a07db050e)